source/javascripts/components/_Variable.js.erb

```diff
@@ -157,6 +157,10 @@
 			return optionGetterSetter(this, "description", newValue);
 		};
 
+		Variable.prototype.shortestDescription = function() {
+			return this.summary() ? this.summary() : this.description();
```

Comment:
@lpusok the description is longest then the summary? (always)

Reply:
This is what you want, because it is still not clear for me.
```
return this.summary().length >  this.description().length ? this.summary() : this.description();
```

Reply:
`return this.description() ? this.description() : this.summary();`
This is good as if description is longer than summary, when both are not empty.

But in the case when they are both empty, we will show empty text. Can we disable the option to click on the expand arrow in that case?